lib/data-types.js

```diff
@@ -345,11 +345,13 @@ inherits(DECIMAL, NUMBER);
 
 DECIMAL.prototype.key = DECIMAL.key = 'DECIMAL';
 DECIMAL.prototype.toSql = function toSql() {
+  let colDefinition = 'DECIMAL';
```

Comment:
Alright, if you rename the varaible to `definition` which is a tad shorter i'm good with this.
Or to result to keep it in line.